### PR TITLE
FEATURE - fetchOrders

### PR DESCRIPTION
**Pull request by dannypaz**
This PR adds `fetchOrders` functionality to the sparkswap ccxt exchange

### CODE REVIEW

**Comment by dannypaz** [comment]
[add fetchOrders to ccxt implementation](https://trello.com/c/u8DqbsEZ/760-add-fetchorders-to-ccxt-implementation)

File: js/sparkswap.js [hunk 1]
```diff
@@ -372,7 +372,68 @@ module.exports = class sparkswap extends Exchange {
...
+                'BID': 'BID',
+                'BUY': 'buy',
+                'SELL': 'sell',
+            },
+            'TYPES': {
+                'LIMIT': 'limit',
+                'MARKET': 'market',
+            },
+        };
+        const res = await this.privateGetV1Orders ({ 'market': symbol });
```
Comment (martineehrlich) [inline js/sparkswap.js]: Do you want to follow the same pattern of:
```
await this.loadMarkets ();

const res = await this.privateGetV1Orders ({ 'market': this.marketId (symbol) });
```

Comment (dannypaz) [reply js/sparkswap.js]: yes! good catch @martineehrlich

**Review by martineehrlich — APPROVED** [review]
LGTM

**Pull request merged**